apps/frontend/src/libs/components/icon-button/icon-button.tsx

```diff
@@ -0,0 +1,41 @@
+import { Icon } from "~/libs/components/components.js";
+
+import { type IconName } from "../icon/types/types.js";
+import styles from "./styles.module.css";
+
+type Properties = {
+	iconColor?: string | undefined;
```

Comment:
Do we need explicitly write type as `string | undefined`? maybe better use just `string` as you do in `Input` component?

Reply:
@liza-veis

Reply:
I can redo using 

color={iconColor ?? ""}

Reply:
> I can also redo it using color={iconColor ?? ""}.

No-no. My question is about what is better:
```ts
type Properties = {
  iconColor?: string;
  ...
}
```
or
```ts
type Properties = {
  iconColor?: string | undefined;
  ...
}
```

Reply:
If icon has `color?: string;` and `iconColor?: string;`, then Icon inside IconButton 
```
Type '{ color: string | undefined; height: number; name: IconName; width: number; }' is not assignable to type 'Properties' with 'exactOptionalPropertyTypes: true'. Consider adding 'undefined' to the types of the target's properties.
  Types of property 'color' are incompatible.
    Type 'string | undefined' is not assignable to type 'string'.
      Type 'undefined' is not assignable to type 'string'.ts(2375)
```

Reply:
Got it